news/2017-09-03-qat-gazette-september-2017.md

```diff
@@ -0,0 +1,57 @@
+
+Fear not however, as **we are currently reviewing applications for a new batch of fresh-faced modders to replenish their ranks!** The latest application process for the osu!taiko, osu!catch and osu!mania! game modes has recently closed, with successful applicants expected to promoted in the coming weeks.
+
+With the discontinuation of Beatmap Nominator tests, new BN applications will consist of an activity check, as well as submission of the applicant’s “best” modding work - either as a standard mod post, or series of suggestions in the moddingv2 panel. 
+
+For those that meet the activity requirements, the QAT will then supplement this with additional random, anonymised examples of the applicant’s modding work from the past three months and evaluate these examples on a number of criteria, such as structure, helpfulness, knowledge of the ranking criteria, and so on. These modding examples will be voted on by the remaining members of the QAT who were not involved in selecting examples of applicant modding, to avoid bias.
+
+**Those with a majority of positive votes will be promoted into one of the available** [Nominator Subdivisions](https://osu.ppy.sh/forum/p/6201585) and closely surveyed on their general attitude (forum posts, in-game chat, modding and icon activity, etc.) to ensure they receive the assistance needed to fulfill their roles as gatekeepers of content in the mapping and modding community.
+
+## Updates to the Quality Assurance Team
+
+With Beatmap Nominator tests no longer a factor, **there have been further reshuffles to role distribution within the QAT.** In addition to having more time available to lead and monitor their respective BN subdivisions, additional roles undertaken by some members are listed below:
+
+- **BN promotions:** Deif, Mao, Nardoxyribonucleic
+- **QAT blog:** JBHyperion, Feerum
+- **Metadata:** IamKwaN, Doyak
+- **Ranking Criteria:** Okorin, pishifat
+- **Beatmap Spotlights:** OnosakiHito, Kurokami
+
+Additionally, please offer a warm farewell to **alacat**, who will be departing from his role in metadata checking for newly qualified beatmaps. Many thanks for your hard work over the years, and we wish you the best of luck in your future endeavours!
+
+## The future of the BNG / QAT
+
+As some of you may be aware, a series of proposals have recently been put forward detailing some **radical changes to the way the Beatmap Nominators and Quality Assurance Team will operate** which we might be seeing in the near future. These include the removal of the osu!standard BN Tier system, introductions of greater rewards and recognition for long-serving and highly-active members, and changes to the workflow and leadership of the QAT.
+
+If you want to check out these proposals and share your opinions, please do so in the [Development forum](https://osu.ppy.sh/forum/t/635507), but remember nothing is confirmed for certain yet, so anything can happen - you have the power to influence our final decisions!
```

Comment:
this links the thread directly and says it links the development forum so i think you could just say check out the proposal and link it

Reply:
Changed link text to "discussion thread"